src/components/addProductsForm/AddProducts.jsx

```diff
@@ -1,5 +1,192 @@
-import React from "react";
+import React, { useState } from "react";
+import { Form, Button, Row, Col, Container } from "react-bootstrap";
+import "./style.scss";
+import db from "../config/firebaseConfig";
 
 export default function AddProducts() {
-  return <div>add Products</div>;
+  const [hasSizes, setHasSizes] = useState(false);
+  const [productFormBrand, setProductFormBrand] = useState("");
+  const [productFormTitle, setProductFormTitle] = useState("");
+  const [productFormDescription, setProductFormDescription] = useState("");
+  const [productFormCategory, setProductFormCategory] = useState("");
+  const [productFormPrice, setProductFormPrice] = useState("");
+
+  // Add products to the database
+  const addNewProductToDatabase = async (e) => {
+    e.preventDefault();
+
+    // Getting all images from the inputs and then pushing them into an array to post the data to the database
+    const allImages = [];
+    const imageInputs = document.getElementsByTagName("input");
```

Comment:
in react, you should never be using document.getElementsByTagName

also this approach doesn't work because, what if this is shown on the page with another input? for example, what if there's a searchbox at the top of the page? this will also get captured

general rule of thumb, we never use `document.` stuff in react except for rare cases

try to search the "react" way to do this :)